collect_app/src/main/res/values/leak_canary_config.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="leak_canary_test_class_name" translatable="false">assertk.Assert</string>
```

Comment:
Could you explain how this fixes things? Might be worth a comment.

Reply:
I've added a comment.

Reply:
Right. Not sure I get why we're using `assertk.Assert` as the test class to detect though? We don't use [assertk](https://github.com/willowtreeapps/assertk) as far as I'm aware

Reply:
> By default, LeakCanary will look for the org.junit.Test class in your classpath and if found, will disable itself to avoid running in tests. However, some apps may ship JUnit in their debug classpaths (for example, when using OkHttp's MockWebServer) so we offer a way to customise the class that is used to determine that the app is running in a test environment.

My understanding is that by default it looks for org.junit.Test and in our case it was found and LeakCanary was disabled.
With this line, it will look for `assertk.Assert` that does not exist so LeakCanary will be enabled.
That's how I understand it but the documentation is poor and I'm not the only one complaining about it.

Reply:
> With this line, it will look for `assertk.Assert` that does not exist so LeakCanary will be enabled.

Right. But that will mean it also runs during tests right? My interpretation of those docs is that we should set `leak_canary_test_class_name` to a class that will only be present in tests (which would be the original intention of `org.junit.Test`.

Reply:
Right, I used `androidx.test.ext.junit.runners.AndroidJUnit4` instead.